<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>933</width>
    <height>557</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>KangarooBPNN</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="RunBtn">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>370</y>
      <width>301</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string>Run</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>291</width>
      <height>201</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>DataSet Para</string>
     </property>
    </widget>
    <widget class="QSplitter" name="splitter_4">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>50</y>
       <width>221</width>
       <height>119</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>DataPatterns</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="DataPatternsEdit"/>
     </widget>
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>TrainDataPatterns</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="TrainDataPatternsEdit"/>
     </widget>
     <widget class="QSplitter" name="splitter_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>TestDataPatterns</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="TestDataPattensEdit"/>
     </widget>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>40</y>
      <width>271</width>
      <height>201</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Network Para</string>
     </property>
    </widget>
    <widget class="QSplitter" name="splitter_29">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>211</width>
       <height>123</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QSplitter" name="splitter_6">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>InputNodes</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="InEdit"/>
     </widget>
     <widget class="QSplitter" name="splitter_7">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>HiddenNodes</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="NeuronEdit"/>
     </widget>
     <widget class="QSplitter" name="splitter_8">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>OutputNodes</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="OutEdit"/>
     </widget>
     <widget class="QSplitter" name="splitter_5">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QLabel" name="label_28">
       <property name="text">
        <string>LearningRate</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="LREdit"/>
     </widget>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>291</width>
      <height>201</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_21">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Train End Para</string>
     </property>
    </widget>
    <widget class="QSplitter" name="splitter_21">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>50</y>
       <width>191</width>
       <height>101</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QSplitter" name="splitter_22">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QLabel" name="label_22">
       <property name="text">
        <string>Epochs</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="EpochsEdit"/>
     </widget>
     <widget class="QSplitter" name="splitter_23">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QLabel" name="label_23">
       <property name="text">
        <string>Accuracy</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="AccuracyEdit"/>
     </widget>
     <widget class="QSplitter" name="splitter_24">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_4">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>280</y>
      <width>271</width>
      <height>201</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_24">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>File Para</string>
     </property>
    </widget>
    <widget class="QSplitter" name="splitter_25">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>231</width>
       <height>123</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QSplitter" name="splitter_26">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QLabel" name="label_25">
       <property name="text">
        <string>InputFilePath</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="InputFileEdit"/>
     </widget>
     <widget class="QSplitter" name="splitter_27">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QLabel" name="label_26">
       <property name="text">
        <string>WSavePath</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="WEdit"/>
     </widget>
     <widget class="QSplitter" name="splitter_28">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QLabel" name="label_27">
       <property name="text">
        <string>VSavePath</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="VEdit"/>
     </widget>
     <widget class="QSplitter" name="splitter_30">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QLabel" name="label_29">
       <property name="text">
        <string>LogPath</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="LogEdit"/>
     </widget>
    </widget>
   </widget>
   <widget class="QTextEdit" name="StatusOutput">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>40</y>
      <width>301</width>
      <height>321</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>933</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
